### ISSUE TITLE
Error On product/translations admin

### ISSUE THREAD

**Issue by asecondwill** [solidusio-contrib/solidus_globalize#92]
I get this error when going to 

/admin/products/ruby-hoodie/translations
```
 Showing /Users/will/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/bundler/gems/solidus_globalize-47c8f90bcf48/app/views/spree/admin/translations/_settings.html.erb where line #17 raised:

undefined local variable or method `preferences' for #<Spree::Store:0x00007fc78164a7f8>
Did you mean?  preference_type
```

**Comment by asecondwill** [comment]
This is on a demo store locally, 
```

source 'https://rubygems.org'
git_source(:github) { |repo| "https://github.com/#{repo}.git" }

ruby '2.6.5'

# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
gem 'rails', '~> 6.0.0'
# Use postgresql as the database for Active Record
gem 'pg', '>= 0.18', '< 2.0'
# Use Puma as the app server
gem 'puma', '~> 3.11'
# Use SCSS for stylesheets
gem 'sass-rails', '~> 5'
# Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker
gem 'webpacker', '~> 4.0'
# Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
gem 'turbolinks', '~> 5'
# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
gem 'jbuilder', '~> 2.7'
# Use Redis adapter to run Action Cable in production
# gem 'redis', '~> 4.0'
# Use Active Model has_secure_password
# gem 'bcrypt', '~> 3.1.7'

# Use Active Storage variant
# gem 'image_processing', '~> 1.2'

# Reduces boot times through caching; required in config/boot.rb
gem 'bootsnap', '>= 1.4.2', require: false

group :development, :test do
  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
end

group :development do
  # Access an interactive console on exception pages or by calling 'console' anywhere in the code.
  gem 'web-console', '>= 3.3.0'
  gem 'listen', '>= 3.0.5', '< 3.2'
  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
  gem 'spring'
  gem 'spring-watcher-listen', '~> 2.0.0'
end

group :test do
  # Adds support for Capybara system testing and selenium driver
  gem 'capybara', '>= 2.15'
  gem 'selenium-webdriver'
  # Easy installation and use of web drivers to run system tests with browsers
  gem 'webdrivers'
end

# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]

gem 'solidus'
gem 'solidus_auth_devise'
gem 'solidus_multi_domain', github: 'solidusio-contrib/solidus_multi_domain', branch: 'master'

gem 'globalize'
gem 'solidus_i18n'
gem 'solidus_globalize',  github: 'solidusio-contrib/solidus_globalize'
```

**Comment by fhirzall** [comment]
@asecondwill have you had any luck working this out? I suspect it might be a version dependency problem with solidus. I have the same exact issue. What version and you running?

**Comment by kennyadsl** [comment]
I'm working on #101 to update this extension to work with the last versions of Solidus. Unfortunately, I think it's not working with Rails 6 yet. I will try to take a look once the main issues will be solved.

**Comment by fhirzall** [comment]
Unsure of rails 6 support, but we resolved the original issue by updating the dependencies. https://github.com/buttercloud/solidus_globalize

**Comment by kennyadsl** [comment]
This is now fixed on master with #101. Also, this extension is now compatible with Rails 6.
